frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@apollo/client": "^3.10.4",
```

Comment:
small change: could u uninstall the packages were not using like apollo/graphql?